lib/tests/fixtures/facts/external/windows/powershell/json.ps1

```diff
@@ -0,0 +1,19 @@
+if (Get-Command ConvertTo-JSON -errorAction SilentlyContinue) {
+    @{
+        'PS1_JSON_FACT1' = 'value1'
+        'ps1_json_fact2' = 2
+        'ps1_json_fact3' = $True
+        'ps1_json_fact4' = @('first', 'second')
+        'ps1_json_fact5' = $Null
+        'ps1_json_fact6' = @{ 'a' = 'b'; 'c' = 'd' }
+    } | ConvertTo-JSON -Depth 1 -Compress)
+} else {
+'{
```

Comment:
Huh, I didn't realize you could do multi-line string literals in PowerShell without making them part of a here-string like:

```powershell
@'
foo
bar
@'
```

Interesting!